src/components/calcite-input/calcite-input.tsx

```diff
@@ -70,6 +71,9 @@ export class CalciteInput {
     this.setDisabledAction();
   }
 
+  /** for number values, displays the locale's group separator */
+  @Prop() displayGroupSeparator = false;
```

Comment:
Maybe we can rename this to just be `separate` or `separated`  to be more consistent with HTML attributes?

Reply:
I feel the name group is necessary here.

Reply:
+1 for the word "group" just because that is the name of the specific character in the Intl.NumberFormat API that is controlled by this prop:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/formatToParts

![image](https://user-images.githubusercontent.com/821864/115303664-be5d3a80-a118-11eb-892b-3cf29d353c4c.png)